pkg/serverless/appsec/httpsec/proxy.go

```diff
@@ -61,6 +61,8 @@ func (lp *ProxyLifecycleProcessor) OnInvokeStart(startDetails *invocationlifecyc
 	eventType := trigger.GetEventType(lowercaseEventPayload)
 	if eventType == trigger.Unknown {
 		log.Debugf("appsec: proxy-lifecycle: Failed to extract event type")
+	} else {
+		log.Debugf("appsec: proxy-lifecycle: Extracted event type: %v", eventType)
```

Comment:
This will be helpful when diagnosing future cases, as this line makes it very easy to rule out the "failed to identify payload" scenario (presence of positive evidence is easier to reason about than absence of negative evidence).

Reply:
Agreed, this is a good addition. Ideally at some point we should add a `String()` method to the event type. Right now, if I recall correctly, it will just print a number.

Reply:
Omg, great minds think alike, I just scrolled down and saw the `String()` method you added! 🎉